eq-author-api/scripts/import-json-to-dynamo.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
The filename is dynamo but the script imports to file system?

Reply:
At the moment it will import to file system for testing but will eventually import to Dynamo

Reply:
Can we not call dynamo yet?

Reply:
Yes ive made a change now to allow this